src/redux/store.ts

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const store = configureStore({
+  reducer: {},
```

Comment:
I would strongly recommend separating out any initial reducer into its own file, and coming up with some placeholder value for the initial state, following the examples here: https://redux.js.org/tutorials/fundamentals/part-4-store

Ideally the initial state is latitude/longitude, so you can hardcode some random place in the US. Then another good step would be to verify that the screen components can read that value from the store, maybe by using alert/console.log on render.

Reply:
I agree! @jamespure I have some files I can share for suggestions if you would like!

Reply:
@lindentree @rweitzman This is redux toolkit. Here a ref to the docs https://redux-toolkit.js.org/usage/usage-guide

You'll add create slices in the features folder and add the slice to the store reducer.

Reply:
Initial state is still a core part of this feature/ticket and should not be left for later. The documentation for store setup isn’t significantly different: https://redux-toolkit.js.org/tutorials/typescript